src/main/java/com/boardgo/domain/review/controller/ReviewController.java

```diff
@@ -41,8 +42,8 @@ public class ReviewController {
     public ResponseEntity<List<ReviewMeetingResponse>> getReviewMeetings(
             @RequestParam("reviewType") ReviewType reviewType) {
         List<ReviewMeetingResponse> reviewMeetings =
-                reviewQueryUseCase.getReviewMeetings(reviewType, currentUserId());
-        if (reviewMeetings.isEmpty()) {
+                reviewQueryFacade.getMeetingsToReview(reviewType, currentUserId());
+        if (Objects.isNull(reviewMeetings)) {
```

Comment:
empty에서 null로 변경하신 이유가 있을까요 ?

Reply:
`getMeetingsToReview` 메소드에서 `reviewType` 에 따라 데이터를 조회하는데,
적합한 `reviewType`이 없으면 결과값으로 null 을 리턴하도록 변경했어요! (기존에는 빈 컬렉션 객체를 반환했는데 굳이 객체를 생성할 필요가 없더라구요)

Reply:
퍼사드에서 조건들이 있는데 조건에 해당하지 않을 때 null로 처리하는 걸로 보여서요..! 
 조건 안에서는 무조건 리스트 안에 결과값이 존재하는건가요?

Reply:
오..! 그러게요! 그걸 빠뜨렸네요!

```suggestion
                reviewQueryFacade.getMeetingsToReview(reviewType, currentUserId());
        if (Objects.isNull(reviewMeetings) || reviewMeetings.isEmpty()) {
```
이렇게 수정하는거 어떤가요?

Reply:
좋습니다!